homeassistant/components/switcher_kis/sensor.py

```diff
@@ -40,92 +42,88 @@ class AttributeDescription:
         unit=POWER_WATT,
         device_class=DEVICE_CLASS_POWER,
         state_class=STATE_CLASS_MEASUREMENT,
-        default_value=0,
     ),
     "electric_current": AttributeDescription(
         name="Electric Current",
         unit=ELECTRICAL_CURRENT_AMPERE,
         device_class=DEVICE_CLASS_CURRENT,
         state_class=STATE_CLASS_MEASUREMENT,
-        default_value=0.0,
     ),
 }
 
 TIME_SENSORS = {
     "remaining_time": AttributeDescription(
         name="Remaining Time",
         icon="mdi:av-timer",
-        default_value="00:00:00",
     ),
-    "auto_off_set": AttributeDescription(
+    "auto_shutdown": AttributeDescription(
```

Comment:
This will change the unique ID right?

Reply:
(and so is a breaking change)

Reply:
Missed it, fixed in upstream lib so this change would not be needed anymore